src/metabase/models/database.clj

```diff
@@ -12,7 +12,13 @@
 
 (i/defentity Database :metabase_database)
 
-(defn- pre-cascade-delete [{:keys [id] :as database}]
+(defn- post-select [{:keys [engine] :as database}]
+  (if-not engine database
+                 (assoc database :features (if-let [driver ((resolve 'metabase.driver/engine->driver) engine)]
+                                             ((resolve 'metabase.driver/features) driver)
```

Comment:
i don't think it's a big deal to send the driver features as part of the actual Database object in api responses in addition to the including that information when we list the `:engines` in the MetabaseBootstrap information.  

they serve different purposes and it's significantly easier to already have the data associated with the Database object when it's returned from api calls.  plus it's a tiny amount of data.